docs/src/pages/components/transitions/SimpleGrow.js

```diff
@@ -26,11 +26,11 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-function Polygon() {
+function Polygon(props) {
```

Comment:
Do we ever use this?

Reply:
i just checked the other demos. the grow demo is the only one that makes a separate component, the others just place the jsx from `Polygon` in the transition component.
Shall I change the grow demo to use the same as Collapse, Fade, Zoom and Slide demos?

Reply:
Sounds good.

Reply:
great, I removed them 👍